k9mail-library/src/main/java/com/fsck/k9/mail/transport/smtp/SmtpTransport.java

```diff
@@ -755,6 +759,64 @@ private String readCommandResponseLine(List<String> results) throws IOException
         return line;
     }
 
+    private void executeCommandPipelined(String format, Object... args) throws IOException {
+        if (format != null) {
```

Comment:
I agree with you and this was copied from executeCommand() and hence I let it as it is .
Though I will change it .